nav2_costmap_2d/include/nav2_costmap_2d/image.hpp

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) 2022 Andrey Ryzhikov
```

Comment:
I would like this file and the image processing files to go into a subdirectory of this directory called `denoise` so that we can separate out sections of the project for that specifically.

Reply:
I [moved](https://github.com/ros-planning/navigation2/pull/2567/commits/8105b531e82305c819208faffc8a55fb7207a81e) `image.hpp`, `image_processing.hpp` to `denoise` (didn't move the test sources to a separate directory, should I?) and did a rebase. Unfortunately, the [CircleCI](https://github.com/ros-planning/navigation2/pull/2567/checks?check_run_id=13097544082) didn't help with the building.